src/applications/burials-v2/components/DeathCertificateUploadMessage.jsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export default function DeathCertificateUploadMessage({ form }) {
+  const isAvailable =
+    form?.burialAllowanceRequested?.service === true ||
+    form?.locationOfDeath?.location !== 'vaMedicalCenter';
+  return isAvailable ? (
+    <>
+      <p className="vads-u-font-size--md vads-u-font-weight--normal vads-u-font-family--sans">
+        Upload a copy of the Veteran’s death certificate including the cause of
+        death.
+      </p>
```

Comment:
Optional feedback - you most likely should be able to just inline the conditional to apply only to the `<p>` tag. This can also be done at a later date

Example:
```jsx
return (
<>
  {isAvailable ? <p>Available content</p> : <p>Not available content</p>}
  <p><strong>How to upload files</strong>
  {/* the rest of the code */}
</>
```